analyzer/src/main/kotlin/managers/PackageJsonUtil.kt

```diff
@@ -34,9 +34,9 @@ internal class PackageJsonUtil {
 
         private data class DefinitionFileInfo(
                 val definitionFile: File,
-                var hasYarnLockfile: Boolean = false,
-                var hasNpmLockfile: Boolean = false,
-                var isYarnWorkspaceRoot: Boolean = false,
+                val hasYarnLockfile: Boolean = false,
```

Comment:
Commit message nit: You should probably say "Assign some properties directly in constructor call" or "Assign most properties directly in constructor call", otherwise it sounds like you were doing all of them, incl. `isYarnWorkspaceSubmodule`.

Reply:
Done.